packages/grant-explorer/src/features/discovery/FilterDropdown.tsx

```diff
@@ -0,0 +1,192 @@
+import { Fragment } from "react";
+import { Link, useSearchParams } from "react-router-dom";
+import { Transition, Listbox } from "@headlessui/react";
+import {
+  ChevronDownIcon,
+  CheckIcon,
+  ChevronUpIcon,
+} from "@heroicons/react/20/solid";
+
+import { CHAINS } from "../api/utils";
+import { Dropdown, DropdownItem } from "../common/Dropdown";
+import { toURL } from "./RoundsFilter";
+
+type Option = {
+  label: string;
+  value: string;
+  children: Option[];
+};
+const filterOptions = [
+  {
+    label: "All",
+    value: "",
+    children: [],
+  },
+  {
+    label: "Type",
+    value: "type",
+    children: [
+      {
+        label: "Quadratic funding",
+        value: "MERKLE",
+      },
+      {
+        label: "Direct grants",
+        value: "DIRECT",
+      },
+    ],
+  },
+  {
+    label: "Status",
+    value: "status",
+    children: [
+      {
+        label: "Active",
+        value: "",
+      },
+      {
+        label: "Taking applications",
+        value: "apply",
+      },
+      {
+        label: "Finished",
+        value: "finished",
+      },
+    ],
+  },
+  {
+    label: "Network",
+    value: "network",
+    children: Object.entries(CHAINS).map(([value, { name }]) => ({
+      label: `Rounds on ${name}`,
+      value,
+    })),
+  },
+] as Option[];
```

Comment:
Let's remove this cast and instead type the variable directly, to reduce the use of casting in general.